keypoints/include/pcl/keypoints/trajkovic_2d.h

```diff
@@ -126,16 +126,31 @@ namespace pcl
       getSecondThreshold () const { return (second_threshold_); }
 
       /** \brief Initialize the scheduler and set the number of threads to use.
-        * \param nr_threads the number of hardware threads to use, 0 for automatic.
+        * \param num_threads the number of hardware threads to use, 0 for automatic.
         */
       inline void
-      setNumberOfThreads (unsigned int nr_threads = 0) { threads_ = nr_threads; }
+      setNumberOfThreads(unsigned int num_threads = 0)
+      {
+#ifdef _OPENMP
+        num_threads_ = num_threads != 0 ? num_threads : omp_get_num_procs();
+#else
+        if (num_threads_ != 1) {
+          PCL_WARN("OpenMP is not available. Setting number of threads to 1\n");
+          num_threads_ = 1;
+        }
```

Comment:
```suggestion
        if (num_threads != 1) {
          PCL_WARN("OpenMP is not available. Setting number of threads to 1\n");
        }
        num_threads_ = 1;
```
Would this not make more sense? To check if the user tries to set the number of threads to something different than 1?